lib/node/nodes/contour.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var Node = require('../node');
+
+var TYPE = 'contour';
+
+var PARAMS = {
+    // the scattered points dataset with real values
+    source: Node.PARAM.NODE(Node.GEOMETRY.POINT),
+    // column of values in source dataset
+    column: Node.PARAM.STRING(),
+    // buffer around my dataset
+    buffer: Node.PARAM.NULLABLE(Node.PARAM.NUMBER(), 0.2),
+    // 0: nearest neighbor, 1: barymetric, 2: IDW
+    method: Node.PARAM.NULLABLE(Node.PARAM.ENUM('nearest_neighbor', 'barymetric', 'IDW'), 'barymetric'),
```

Comment:
We use these literals twice. We should create a `variable/constant` and use it instead.